scripts/lib/utilities.js

```diff
@@ -4,20 +4,7 @@
 var fs = require('fs');
 var path = require("path");
 
-fs.ensureDirSync = function (dir) {
```

Comment:
@soumak77 Why did you move this into the after_prepare.js script.? From what I can tell, it is referenced in the utilities.js file but not in after_prepare.js.  Wasn't sure if I was missing something

Reply:
Good catch.  Caused by copying old file contents.